plexus-compilers/plexus-compiler-javac/src/main/java/org/codehaus/plexus/compiler/javac/JavaxToolsCompiler.java

```diff
@@ -36,15 +36,16 @@
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.Locale;
 import java.util.concurrent.CopyOnWriteArrayList;
 
 /**
  * @author Olivier Lamy
  * @author <a href="mailto:david.lloyd@redhat.com">David M. Lloyd</a>
  * @since 2.0
  */
-@Component( role = Compiler.class )
-public class JavaxToolsCompiler implements InProcessCompiler
+@Component( role = InProcessCompiler.class )
```

Comment:
not a change but fix a bug to be able to use dependency injection for this...
it was simply wrong for very long but the only usage was `new JavaxToolsCompiler()` :)